src/apps/applets/crud/applets_history.py

```diff
@@ -77,3 +77,11 @@ async def set_report_configuration(
         query = query.values(**schema.dict(by_alias=False))
 
         await self._execute(query)
+
+    async def get_current_versions_by_applet_id(self, applet_id: uuid.UUID) -> str:
```

Comment:
You can reuse something instead of creating new method. Additionally, current version stored in `applets` table